### DISCUSSION TITLE
Review the latests additions to badges

### DISCUSSION THREAD

**Discussion by smileBeda** [community/community#18477] [Other Feature Feedback, Questions, & Ideas]
I would like to bring this to your attention:
https://github.community/t/new-github-badges-released-awesome/255454/4

The badges should not show any repo. It should for starters not show private repos as those can be just used to produce hundreds, thousands of fake commits. It should pull the biggest repos contributed to, not the smallest or own ones.

See also for details https://github.community/t/new-github-badges-released-awesome/255454/4?u=tukutoi and the subsequent voices expressing the same concern.

Thank You!

**Comment by proarafat** [comment]
@TukuToi  Yup your idea is good.
You can review that on Github Support. Because on Github Support there have a Review Feature. To review the any Functions and projects of Github. https://support.github.com/
And also plz check my Comment as a Answer. Plz plz (if that answer)

**Reply by smileBeda** [reply]
So it gets a badge of problem solver?
Com-on... 
I came here from another forum, where it was suggested to go here... I won't go again to another forum
Also, if you think the idea is so good, probably the right thing is to at least add a "+1", but hey, right, that does not covert to badges.... I get it.

If they (they being GitHub developers, or Microsoft, for that is) want to convert GitHub into a kindergarten of fake badges, like the one above attempts to get, so be it. I will be happy on other platforms too, and so other, less game-addict users.

**Reply by proarafat** [reply]
No no i just suggest to review that. Now that's your decissition .  That's not requered.

**Comment by leeeunlin** [comment]
I agree with you
